public/locales/en-EN/glossary.json

```
@@ -0,0 +1,6 @@
+{
+  "navbar.tour-selection": "Tour selection",
+  "support-time": "Daily from 9:00 to 20:00",
```

"support-time": "Daily from 9:00 to 20:00",
```suggestion
  "support-time": "Daily from 9 A.M. to 8 P.M.",
```